ATM.py

```diff
@@ -0,0 +1,141 @@
+__author__ = 'Adi Kadar Levi'
+
+
+class ATM:
+    def __init__(self):
+        self.id_num = 0
+        self.password = 0
+        self.balance = 0
+        self.__pass_dict = {}
+        self.__bal_dict = {}
+
+    def get_passwords_dictionary(self):
+        bank_data_path = r'C:\Users\Elior Mor Yosef\Documents\GitHub' \
+                         r'\python-training\python-training\bank.txt'
+
+        with open(bank_data_path, 'r') as bank_data:
+            for line in bank_data:
+                words = line.split()
+                self.__pass_dict[words[0]] = words[1]
+        return self.__pass_dict
+
+    def get_balance_dictionary(self):
+        bank_data_path = r'C:\Users\Elior Mor Yosef\Documents\GitHub' \
+                         r'\python-training\python-training\bank.txt'
+
+        with open(bank_data_path, 'r') as bank_data:
+            for line in bank_data:
+                words = line.split()
+                self.__bal_dict[words[0]] = words[2]
+        return self.__bal_dict
+
+    def check_id(self, id_input):
+        if id_input not in self.get_passwords_dictionary().keys():
```

Comment:
you don't have to read the data from the file each time...
you already put the data in a dictionary of the class, read the data one time in the contractor.